src/cgame/cg_consolecmds.c

```diff
@@ -2973,22 +2973,56 @@ static void CG_EditHudComponentComplete(void)
 
 #define EDITCOMPONENT_CROSSHAIR_STRING "editcomponent crosshair"
 
-static void CG_CrosshairSize_f(void)
+static void CG_CrosshairSizePos_f(void)
 {
 	if (trap_Argc() > 1)
 	{
-		const char *token;
-
-		token = CG_Argv(1);
+		const char *token = CG_Argv(1);
 
 		if (Q_isanumber(token))
 		{
-			float size = Q_atof(token);
+			const float value = Q_atof(token);
+
+			float sizeX = cg_crosshairSize.value;
+			float sizeY = cg_crosshairSize.value;
+
+			float scaleX = cg_crosshairScaleX.value;
+			float scaleY = cg_crosshairScaleY.value;
+
+			float offsetX = cg_crosshairX.value;
+			float offsetY = cg_crosshairY.value;
+
+			const char *cmd = CG_Argv(0);
+
+			if (!Q_stricmp(cmd, "cg_crosshairSize_f"))
+			{
+				sizeX = sizeY = value;
+			}
+			else if (!Q_stricmp(cmd, "cg_crosshairX_f"))
+			{
+				offsetX = value;
+			}
+			else if (!Q_stricmp(cmd, "cg_crosshairY_f"))
+			{
+				offsetY = value;
+			}
+			else if (!Q_stricmp(cmd, "cg_crosshairScaleX_f"))
+			{
+				scaleX = value;
+			}
+			else if (!Q_stricmp(cmd, "cg_crosshairScaleY_f"))
+			{
+				scaleY = value;
+			}
```

Comment:
Rather than calling the same function here for multiple cvars, we could store all these in a struct and update values in that, and then just recompute the position with those values whenever they're modified, let me know if you want me to refactor this like that.

Initially I added separate functions for position/scale cvars, but that caused update collisions with `vid_restart`/map load, where the position/size updates would come from both this and the position/scale functions, which reset some of the values depending on which was called first.